samples/bluetooth/hci_spi/boards/nrf52dk_nrf52832.overlay

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
```

Comment:
I always have to dig through my notes to find the corresponding host side.

It would be much appreciated if you could add a section to the readme with the steps required to attach the zephyr host to this `nrf52dk_nrf52832` board flashed with the `hci_spi` sample. Ie a dts snippet + kconfig.